src/System.Collections.NonGeneric/src/System/Collections/SortedList.cs

```diff
@@ -455,8 +454,7 @@ public virtual Object GetKey(int index)
         // 
         public virtual IList GetKeyList()
         {
-            if (_keyList == null) _keyList = new KeyList(this);
-            return _keyList;
+            return _keyList ?? (_keyList = new KeyList(this));
```

Comment:
Please revert this change but do feel free to put this if block on its own line to make it match the coding guidelines.